### ISSUE TITLE
Data-linking to properties in nested objects

### ISSUE THREAD

**Issue by virajsanghvi** [BorisMoore/jsviews#101]
I looked around, but didn't see any duplicates of this question. I suspect what I'm asking for is not possible, but thought others would run into this and having an answer would be nice.

Let's say you have the template:

``` html
 <div id="holder"></div>

  <script id="test" type="text/x-jsrender">
    <span data-link="html{:(foo && foo.subfoo && foo.subfoo.subsubfoo) ? foo.subfoo.subsubfoo : 'n/a'}"></span>
  </script>
```

and javascript:

``` javascript
    $.templates({
      test: "#test"
    });

    var o = {
      foo: {}
    };

    $.link.test('#holder', o);

    // This reflects itself in the ui (i.e. #holder contains 1)
    $.observable(o).setProperty('foo.subfoo', { subsubfoo: 1 });

    // This doesn't ((i.e. #holder still contains 1)
    $.observable(o.foo.subfoo).setProperty('subsubfoo', 3);
```

As you can see in the template, I've conditionally printed subsubfoo because I'd otherwise get an error that the parent doesn't exist. I figured by checking all the parents, at least one would receive change events and trigger the data-link, but it doesn't seem to happen when the object already exists in the second call to update the value to 3. 

Basically, I'm wondering if you have any methods to get something like this to work (or to be able to write the data-link for a property whose parent object doesn't exist yet). If not, is it possible to have change events bubble up their parents to trigger and data-links for the ternary operator in this example template (I'm guessing there's a perf cost to this)? 

Anyways, thanks in advance, and thanks for building these libraries!
